deploy/crds/org_v1_che_crd.yaml

```diff
@@ -507,6 +507,11 @@ spec:
                     TLS mode. This is enabled by default. Disabling TLS may cause
                     malfunction of some Che components.
                   type: boolean
+                useServiceHostNames:
```

Comment:
If you google hostname - it will be one word
```suggestion
                useServiceHostnames:
```

Reply:
or just `serviceHostnames` ?

Reply:
As for me both variants are Ok:  serviceHostnames and useServiceHostnames